transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/jobscheduling/AlarmManagerScheduler.java

```diff
@@ -44,12 +47,15 @@
 
   private final SchedulerConfig config;
 
+  private final Clock clock;
+
   public AlarmManagerScheduler(
       Context applicationContext, EventStore eventStore, SchedulerConfig config) {
     this(
         applicationContext,
         eventStore,
         (AlarmManager) applicationContext.getSystemService(Context.ALARM_SERVICE),
+        new WallTimeClock(),
```

Comment:
Also, for consistency, can you inject this clock via the `SchedulingModule`?

And actually, since we are using ELAPSED_REALTIME, you want the `@Monotonic Clock` not the Wall Time one.

Reply:
PTAL.